velox/exec/fuzzer/PrestoSql.cpp

```diff
@@ -309,6 +309,33 @@ std::string toCallSql(const core::CallTypedExprPtr& call) {
     sql << "[";
     toCallInputsSql({call->inputs()[1]}, sql);
     sql << "]";
+  } else if (call->name() == "switch") {
+    VELOX_CHECK_GE(
+        call->inputs().size(),
+        2,
+        "Expected at least two arguments to function 'switch'");
+    // If two inputs, handle like an "if"
+    if (call->inputs().size() == 2) {
+      sql << "if(";
+      toCallInputsSql({call->inputs()[0]}, sql);
+      sql << ", ";
+      toCallInputsSql({call->inputs()[1]}, sql);
+      sql << ")";
+    } else {
```

Comment:
This special case doesn't seem to be needed. For switch with two inputs, we can still convert it to "case when C then B1 end".